src/gateways/Gateway.php

```diff
@@ -313,9 +313,6 @@ public function refund(Transaction $transaction): RequestResponseInterface
         $request = $this->createRequest($transaction);
         $refundRequest = $this->prepareRefundRequest($request, $transaction->reference);
 
-// HACK! Work-around to always return a successful refund response. We'll refund externally and then this would ensure Craft is in sync.
-return new RefundResponse($refundRequest);
-
         $processRefund = $this->performRequest($refundRequest, $transaction);
```

Comment:
@AndriiBalaban - instead of manually deleting the temporary work-around, I think it's better to revert it using git. That way, it would also remove the unneeded "use" and the "RefundResponse" file. 
It's both cleaner (no left-over files/code lines) and clearer when looking at the git history.